src/v/redpanda/admin/server.cc

```diff
@@ -2407,7 +2407,7 @@ void admin_server::register_features_routes() {
           ss::httpd::features_json::license_response res;
           res.loaded = false;
           const auto& ft = _controller->get_feature_table().local();
-          const auto& license = ft.get_license();
+          const auto& license = ft.get_license(false);
```

Comment:
we do want to return the evaluation license via the `get_license` endpoint

Reply:
sort of related to this - the evaluation license has to supersede all functional effects of license enforcement, but

1. should the response from `GET /v1/features/enterprise` look like there's a real license installed? or 
2. should we suppress the license nag during the trial period?
3. should the `has_valid_license` bool in phone-home telemetry be an enum? or maybe we add an additional field for "evaluation_period"?

Reply:
IMO yes, the evaluation period should work as if a real trial license had been installed (i.e., don't nag until expiry, show that there is a trial license installed, expiry time metric > 0).

```
% rpk cluster license info
LICENSE INFORMATION
===================
License status:     valid
License violation:  false
Organization:       Redpanda Built-In Evaluation Period
Type:               free_trial
Expires:            Dec 6 2024
```

> should the has_valid_license bool in phone-home telemetry be an enum? or maybe we add an additional field for "evaluation_period"?

Probably easiest to add a new field for "evaluation_period". I guess this could be derived from the license hash (`id_hash == 0`) but probably best to add it explicitly.

Reply:
Aha, totally missed the free trial  type on the license!

Reply:
The request from product is to have it return "trial" just like if it's a generated trial license.  We should be able to tell that it's the evaluation by:
* The organization, and
* The empty SHA256 checksum